src/perf_counters.h

```diff
@@ -49,13 +49,15 @@ namespace internal {
 // operator[]) of this object.
 class PerfCounterValues {
  public:
-  explicit PerfCounterValues(size_t nr_counters) : nr_counters_(nr_counters) {
-    BM_CHECK_LE(nr_counters_, kMaxCounters);
+  explicit PerfCounterValues(size_t nr_counters)
```

Comment:
`int` please instead of `size_t`.

Reply:
Just to be clear, the original code in the `main` branch opted for this:

https://github.com/google/benchmark/blob/60b16f11a30146ac825b7d99be0b9887c24b254a/src/perf_counters.h#L52

Do you want me to change here as a cleanup side-effect in this PR?

Reply:
if you don't mind, that would be great.  if it causes problems we can do it (i can) as a followup.